docs/specification-ligo.md

```diff
@@ -898,6 +899,25 @@ Parameter (in Michelson):
 - a `string`: the custom entrypoint name stored inside `custom_entrypoints` to execute
 - a `bytes`: the `packed` representation of the `<ep_param>` to execute it with
 
+### **freeze**
+
+```ocaml
+type freeze_param = nat
+```
+
```

Comment:
Need to add the Michelson representation too.

Reply:
Have added.